redi-connect-front/public/index.html

```diff
@@ -1,29 +1,24 @@
 <!DOCTYPE html>
```

Comment:
What have been really changed in this file?

Reply:
A bit hard to see because the links have basically just been shifted around but I added the CDN links for the carousel in line 10 and 12.

Reply:
Just seeing this. Why don't we instead use `@use "<the url here>"` in `Carousel.scss`? Makes dependencies much easier to follow.

Note: [the @import rule is being deprecated](https://sass-lang.com/documentation/at-rules/import)

While we're on the subject: let's consider creating a ticket to move the other stylesheets in this `<head>` into more appropriate .scss files.

Reply:
Ok, this is not the way we want to go. The files should not be referenced like this since then it will be loaded always and everywhere. Make sure to use the first suggested version for referencing the styles (https://www.npmjs.com/package/react-slick). 
Another tip, next time when you see that in the file there are a lot of formatting changes write a comment on the file, this way it will save use time at reviewing to identify what to look for.

Reply:
After talking to @ericbolikowski we agreed on creating a separate story for replacing the imports with `use`.

imports are now within the `Carousel.scss` file and have been delated from the `index.html`